framework/validators/ExistValidator.php

```diff
@@ -73,6 +73,12 @@ class ExistValidator extends Validator
      */
     public $targetAttributeJunction = 'and';
 
+    /**
+     * @var bool whether this validator can run on salve connections
+     * @since 2.0.14
+     */
+    public $canValidateOnSlaveDb =  false;
```

Comment:
`$useMasterDb`? https://github.com/yiisoft/yii2/pull/14424#discussion_r127182082

Reply:
I'd say `$forceMasterDb`.

Reply:
changed to $forceMasterDb and take useMaster to run on master db.